Extension/src/platform.ts

```diff
@@ -14,6 +14,16 @@ import * as nls from 'vscode-nls';
 nls.config({ messageFormat: nls.MessageFormat.bundle, bundleFormat: nls.BundleFormat.standalone })();
 const localize: nls.LocalizeFunc = nls.loadMessageBundle();
 
+export function GetOsName(processPlatform: string | undefined): string | undefined {
+    switch (processPlatform) {
+        case "win32": return "Windows";
+        case "darwin": return "MacOS";
+        case "linux32": return "Linux(32 bit)";
+        case "linux": return "Linux(64 bit)";
```

Comment:
Should be a space before (. Or you could remove the 32-bit Linux stuff and just use "Linux".